### PR TITLE
[NodeBundle] Throw not found exception if node is not found

### PR DESCRIPTION
**Pull request by dannyvw**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #3264

### CODE REVIEW

File: src/Kunstmaan/NodeBundle/Controller/NodeAdminController.php [hunk 1]
```diff
@@ -822,6 +822,9 @@ public function editAction(Request $request, $id, $subaction): Response
         $this->init($request);
         /* @var Node $node */
         $node = $this->em->getRepository(Node::class)->find($id);
+        if (!$node) {
+            throw $this->createNotFoundException('No node found for id ' . $id);
```
Comment (acrobat) [inline src/Kunstmaan/NodeBundle/Controller/NodeAdminController.php]:
```suggestion
            throw $this->createNotFoundException(sprintf('No node found for id "%s"', $id));
```

**Review by acrobat — CHANGES_REQUESTED** [review]
A small comment otherwise go to merge!

File: src/Kunstmaan/NodeBundle/Controller/NodeAdminController.php (hunk 1 above)
Comment (dannyvw) [reply src/Kunstmaan/NodeBundle/Controller/NodeAdminController.php]: @acrobat Done

**Review by acrobat — APPROVED** [review]

**Comment by acrobat** [comment]
Thanks @dannyvw!

**Pull request merged**